quickjs.c

```diff
@@ -53043,6 +53043,33 @@ JSValue JS_NewArrayBufferCopy(JSContext *ctx, const uint8_t *buf, size_t len)
                                         TRUE);
 }
 
+/* return -1 if exception (proxy case) or TRUE/FALSE */
+int JS_IsArrayBuffer(JSContext *ctx, JSValueConst val)
+{
+    JSObject *p;
+
+    if (JS_VALUE_GET_TAG(val) == JS_TAG_OBJECT) {
```

Comment:
Wouldn't it make more sense to check for the negated condition here and return false early? You'd save an indent level.

Reply:
Agree. Initially I just wanted to preserve the style taken from `JS_IsArray()`.